go.mod

```diff
@@ -18,10 +18,12 @@ require (
 	github.com/go-text/typesetting v0.1.0
 	github.com/godbus/dbus/v5 v5.1.0
 	github.com/jackmordaunt/icns/v2 v2.2.6
+	github.com/jeandeaual/go-locale v0.0.0-20220711133428-7de61946b173
```

Comment:
Isn't it a bit worrying that the package hasn't been updated for two years and imports Fyne v1?

Reply:
It's the type of library where it's coded once and it just works so that aspect does not worry me.
I could offer them a v2 upgrade for Fyne and see where it goes?

Reply:
I think that would be good. We don't want to compile two versions of Fyne into each application 😅